bgpd/bgp_vty.c

```diff
@@ -17,7 +17,6 @@
  * with this program; see the file COPYING; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
  */
-
```

Comment:
please remove the unnecessary white space change

Reply:
done